src/frontend/admin/src/components/templates/organizations/form/OrganizationForm.tsx

```diff
@@ -159,6 +163,28 @@ export function OrganizationForm(props: Props) {
                     )}
                   />
                 </Grid>
+                <Grid xs={12}>
+                  {!organizationQuery.countries && (
+                    <Box
+                      sx={{
+                        display: "flex",
+                        justifyContent: "center",
+                        alignItems: "center",
+                      }}
+                    >
+                      <CircularProgress size={30} />
+                    </Box>
+                  )}
+                  {organizationQuery.countries && (
+                    <RHFSelect
+                      name="country"
+                      options={organizationQuery.countries ?? []}
+                      label={intl.formatMessage(
+                        organizationFormMessages.countryLabel,
+                      )}
+                    />
+                  )}
```

Comment:
Maybe we could simply disable the select if options are not yet ready what do you think ?